### PR TITLE
fix(core): Fix payload property in `workflow-post-execute` event

### PR DESCRIPTION
**Pull request by ivov**
The `success` property in the payload for the `n8n.workflow.success` and `n8n.workflow.failed` log streaming events does not refer to whether the status was `'success'` but rather refers to whether the execution `finished` or not. This detail was misinterpreted when the event bus was decoupled from the event bus.

Ref. https://github.com/n8n-io/n8n/pull/9724/files#diff-a91344c72b2b4a696fa0b18ef1a154dad6d7594baba2285265e37b2695c4f9edL401

https://share.cleanshot.com/MKlnkCNW

https://linear.app/n8n/issue/PAY-1841

### CODE REVIEW

File: packages/cli/src/events/log-streaming-event-relay.ts [hunk 1]
```diff
@@ -112,7 +112,7 @@ export class LogStreamingEventRelay extends EventRelay {
 
 		const payload = {
 			...rest,
-			success: runData?.status === 'success',
+			success: !!runData?.finished, // despite the `success` name, this reports `finished` state
```
Comment (tomi) [inline packages/cli/src/events/log-streaming-event-relay.ts:115]: 👍 for documenting the oddity

**Review by tomi — APPROVED** [review]
LGTM 🚀

**Comment by ivov** [comment]
Synced with Marcus and customer would be fine to do a normal upgrade, so removing `release/backport`

**Comment by janober** [comment]
Got released with [`n8n@1.54.3`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.54.3)

**Pull request merged**